tools/demikernel_ci.py

```diff
@@ -314,6 +453,45 @@ def run_pipeline(
         move(log_directory, old_dir)
     mkdir(log_directory)
 
+    if libos == "catnapw":
```

Comment:
Do we still need this check? We have already changed to `catnap` at other places in the rust codebase.

Reply:
Yes, we do in `demikernel_ci.py`. I intend to address this in future PRs, because I understand that integrating Windows ASAP is more important than addressing naming schemes. At the same time I intend to address duplicate `_windows` fuctions.

Reply:
Ok, thanks!